examples/reference/elements/bokeh/Tiles.ipynb

```diff
@@ -92,11 +100,24 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.8"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
 }
```

Comment:
I should've commented below - my mistake. But this is the metadata I was unable to remove via following command. Would love to know what I'm doing wrong and I can manually fix this if needed.

```
jupyter nbconvert --clear-output --ClearMetadataPreprocessor.clear_notebook_metadata=True --ClearMetadataPreprocessor.clear_cell_metadata=True Tiles.ipynb
```